x/json.go

```diff
@@ -0,0 +1,52 @@
+package x
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	jsonpatch "github.com/evanphx/json-patch"
+
+	"github.com/ory/x/jsonx"
+)
+
+// Applies a JSON Patch to an object while checking for denied paths in the patch
+//
+// If the patch contains an update to a field matching a denied path,
+// the whole patch is "rejected" and an error is returned
+func ApplyJSONPatch(p json.RawMessage, object interface{}, denyPaths ...string) error {
+	patch, err := jsonpatch.DecodePatch(p)
+	if err != nil {
+		return err
+	}
```

Comment:
Why do we copy this file from the `ory/x` repository?

https://github.com/ory/x/blob/master/jsonx/patch.go

Is it not the same?

Reply:
You are absolutely right, I just looked at hydra and copied it from there. I did have a few modifications in mind, but it ended up working without it. I removed the files.